packages/gatsby-theme-docs/src/layouts/internals/layout-page-release-notes-filters.js

```diff
@@ -41,22 +42,43 @@ const GridContainer = styled.div`
   @media screen and (${designSystem.dimensions.viewports.largeTablet}) {
     display: block;
     grid-area: page-navigation;
-    width: ${designSystem.dimensions.widths.pageNavigation};
-  }
-  @media screen and (${designSystem.dimensions.viewports.laptop}) {
-    width: ${designSystem.dimensions.widths.pageNavigationSmall};
-  }
-  @media screen and (${designSystem.dimensions.viewports.desktop}) {
-    width: ${designSystem.dimensions.widths.pageNavigation};
   }
 `;
-const StickyContainer = styled.div`
-  position: sticky;
-  top: ${designSystem.dimensions.spacings.xxl};
-  margin: 0 0 ${designSystem.dimensions.spacings.s};
-  padding: 0 ${designSystem.dimensions.spacings.m}
-    ${designSystem.dimensions.spacings.m};
-`;
+
+const stickyContainerCss = css({
+  position: 'sticky',
+  top: 0,
+  marginTop: `-1px`,
+  padding: `1px ${designSystem.dimensions.spacings.m} ${designSystem.dimensions.spacings.m}`,
+});
+const stuckContainerCss = css(stickyContainerCss, {
+  maxHeight: '100vh',
+  overflow: 'auto',
+  scrollbarWidth: 'thin',
```

Comment:
This is only supported in FF. Do we really need it?

https://caniuse.com/?search=scrollbar-width

Reply:
I tested it now without it and I think we don't need it.